### PR TITLE
FastFT bug + failing test

### PR DESCRIPTION
**Pull request by matthew-andrews**

### CODE REVIEW

**Comment by matthew-andrews** [comment]
@ironsidevsquincy

File: server/controllers/capi.js [hunk 1]
```diff
@@ -130,7 +138,7 @@ module.exports = function(req, res, next) {
 									};
 								}).get(),
 								showTOC: res.locals.flags.articleTOC.isSwitchedOn && $subheaders.length > 2,
-								isColumnist: articleV1.item.metadata.primarySection.term.name === 'Columnists',
+								isColumnist: isColumnist,
```
Comment (ironsidevsquincy) [inline server/controllers/capi.js]: naively assumed there would always be a primarySection...

Comment (ironsidevsquincy) [inline server/controllers/capi.js]: actually that might be the case, issue here was no `articleV1` - probably better to safe though

Comment (matthew-andrews) [inline server/controllers/capi.js]: ah.  that is probably a correct assumption.  The false assumption was that the content would always be in CAPI v1 ;-)

http://api.ft.com/content/items/v1/b002e5ee-3096-3f51-9925-32b157740c98?apiKey=XXX => 404
http://api.ft.com/content/b002e5ee-3096-3f51-9925-32b157740c98?apiKey=XXX => 200

Comment (matthew-andrews) [inline server/controllers/capi.js]: could probably simplify the check to just be `if (articleV1)` to be honest… would you like me to do this?

Comment (ironsidevsquincy) [inline server/controllers/capi.js]: if that assumption (always a primarySection) correct, than yeah, might as well simplify (I might do the same to https://github.com/Financial-Times/next-grumman/blob/master/server/transforms/byline.js#L4)

**Pull request merged**